src/olympia/addons/serializers.py

```diff
@@ -231,8 +233,8 @@ def fake_object(self, data):
 
 class LicenseSerializer(serializers.ModelSerializer):
     is_custom = serializers.SerializerMethodField()
-    name = LicenseNameSerializerField()
-    text = TranslationSerializerField()
+    name = LicenseNameSerializerField(allow_null=True)
+    text = TranslationSerializerField(allow_null=True)
```

Comment:
These were an edge case in how we use `LicenseSerializer` for custom licences - in effect the instance was seen as a `None` for an update too - so providing the `kwarg` just changes it back to the previous behavior.